core/src/main/java/org/bitcoinj/core/PeerGroup.java

```diff
@@ -243,6 +243,8 @@ public void onPeerDisconnected(Peer peer, int peerCount) {
     /** The default timeout between when a connection attempt begins and version message exchange completes */
     public static final int DEFAULT_CONNECT_TIMEOUT_MILLIS = 5000;
     private volatile int vConnectTimeoutMillis = DEFAULT_CONNECT_TIMEOUT_MILLIS;
+    
+    private boolean bloomFilteringEnabled = true;
```

Comment:
Oh yes, in bitcoinj volatile fields start with a v, as in vBloomFilteringEnabled. Just a bit of hungarian notation to help us see what's safe and what's not when reading the code.